ocs_ci/framework/__init__.py

```diff
@@ -330,5 +330,27 @@ def current_cluster_name(self):
         """
         return self.ENV_DATA.get("cluster_name")
 
+    def is_provider_exist(self):
+        """
+        Check if the provider cluster exists in the clusters
+
+        Returns:
+            bool: True, if the provider cluster exists in the clusters. False, otherwise.
+
+        """
+        cluster_types = [cluster.ENV_DATA["cluster_type"] for cluster in self.clusters]
+        return "provider" in cluster_types
```

Comment:
I'm thinking if it will not be little bit more readable with using `any(...)`, something like this:

```python
return any([cluster.ENV_DATA["cluster_type"] == "provider" for cluster in self.clusters])
```
But I'm ok with leaving as it is.